src/routes/HomePrivate.js

```diff
@@ -1,9 +1,11 @@
 import React from 'react'
 import GetStarted from '../components/GetStarted'
 import PendingTradeRequests from '../components/PendingTradeRequests'
+import StatusTradeRequests from '../components/StatusTradeRequests'
```

Comment:
The component is imported, but it's not being used ... looks the changes in this file are not necessary.